### PR TITLE
docs(main-readme): update readme

### PR DESCRIPTION
**Pull request by NikolaM-Dev**
description of moldyup and elimination of things that do not correspond to the installer project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,4 +96,4 @@ Moldy is on the following social networks
 
 ---
 
-Made with ❤️ in the world
+<div align="center">Made with ❤️ in the world</div>
```
Comment (TeoDev1611) [inline README.md]: Here make not center the text only in the left side 😄

**Review by TeoDev1611 — CHANGES_REQUESTED** [review]
Add this for merge please

File: README.md (hunk 1 above)
Comment (TeoDev1611) [reply README.md]: And add the license badge

Comment (NikolaM-Dev) [reply README.md]: with this should be to lick it

**Review by TeoDev1611 — APPROVED** [review]
Thanks 👍

**Pull request merged**